Source/JavaScriptCore/API/JSWrapperMap.mm

```diff
@@ -512,9 +514,9 @@ - (ConstructorPrototypePair)allocateConstructorAndPrototypeInContext:(JSContext
         putNonEnumerable(context, constructor, @"prototype", prototype);
 
         Protocol *exportProtocol = getJSExportProtocol();
-        forEachProtocolImplementingProtocol(m_class, exportProtocol, ^(Protocol *protocol, bool&){
-            copyPrototypeProperties(context, m_class, protocol, prototype);
-            copyMethodsToObject(context, m_class, protocol, NO, constructor);
+        forEachProtocolImplementingProtocol(self->m_class, exportProtocol, ^(Protocol *protocol, bool &) {
+            copyPrototypeProperties(context, self->m_class, protocol, prototype);
+            copyMethodsToObject(context, self->m_class, protocol, NO, constructor);
```

Comment:
What motivates this change? The `self->` should not be needed. And adding a space in `bool &` is simply changing away from WebKit coding style without giving a reason.